src/main.rs

```diff
@@ -75,6 +75,10 @@ struct Cli {
     #[arg(long, default_value_t = 10, conflicts_with = "input")]
     gga_period: u64,
 
+    /// Send the GGA in the HTTP header
+    #[arg(long)]
+    gga_header: bool,
```

Comment:
Not sure if `gga_header` it's the correct name, as this could be either gga or cra based on the provided parameters, maybe `send_as_header`?

Reply:
I think gga_header and gga_period could both probably be changed to `nmea_*` or `nmea_sentence_*` (but that's kind of long) - GGAs and CRAs are both nmea sentences and the flags work for both